hw/top_earlgrey/rdc/chip_earlgrey_asic_scenario.tcl

```diff
@@ -4,8 +4,63 @@
 #
 # Meridian RDC Reset Scenario
 
+###############
+# Reset Group #
+###############
+
+# List of external reset group
+set ext_rst_lst {}
+# POR_N active low
+lappend ext_rst_lst [list { POR_N } {reset low}]
+
+foreach_in_collection j [get_pins top_earlgrey.rstmgr_aon_resets.*] {
```

Comment:
o wow, do we need to do this to inform the tool where all the resets are?

Reply:
The tool figured out the POR_N and SW reset signals (+ SPI_DEVICE FIFO resets). I need to check if these are all resets or not.

Reply:
o those definitely are.